uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/controller/UniProtKBSearchControllerIT.java

```diff
@@ -2009,6 +2009,34 @@ void searchXrefFullForSingleIdXrefAndJsonFormatReturnBadRequest() throws Excepti
                                         "Fields 'xref_ensembl_full, xref_kegg_full' are only supported by TSV (text/plain;format=tsv) format.")));
     }
 
+    @Test
+    void searchWhitelistVGNCIdDefaultSearch() throws Exception {
```

Comment:
We already have 2 tests covering this logic: 
searchWhiteListPRODatabaseFieldDefaultSearch and searchWhiteListHGNCDatabaseFieldDefaultSearch

Maybe we can have a test with an invalid prefix?